dangerfile.js

```diff
@@ -1,4 +1,4 @@
-// eslint-disable-next-line import/no-extraneous-dependencies, no-unused-vars
+// eslint-disable-next-line import/no-extraneous-dependencies, no-unused-vars, object-curly-newline
```

Comment:
Should we just do the obj new line instead of disabling? I imagine it wants:
```
import {
   danger,
   warn,
   fail,
   message
} from 'danger';
```